src/app/shared/NoteActions.svelte

```diff
@@ -150,31 +171,50 @@
       return 0
     }, handler.event.tags)
 
-    const entity = last(templateTag) === "note" ? nip19.noteEncode(note.id) : nevent
+    const entity = last(templateTag) === "note" ? nip19.noteEncode(event.id) : nevent
 
     window.open(templateTag[1].replace("<bech32>", entity))
   }
 
+  const context = deriveEvents(repository, {filters: getReplyFilters([event])})
+
+  $: children = $context.filter(e => isChildOf(e, event))
+
   let view
   let actions = []
   let handlersShown = false
 
+  $: lnurl = getLnUrl(event.tags?.find(nthEq(0, "zap"))?.[1] || "")
+  $: zapper = lnurl ? deriveZapper(lnurl) : deriveZapperForPubkey(event.pubkey)
+  $: muted = $userMutes.has(event.id)
+  // Split out likes, uniqify by pubkey since a like can be duplicated across groups
+  $: likes = uniqBy(prop("pubkey"), children.filter(isLike))
+
+  // Split out zaps
+  $: zaps = children
+    .filter(e => e.kind === 9735)
+    .map(e => ($zapper ? zapFromEvent(e, $zapper) : null))
+    .filter(identity)
+  $: replies = sortEventsDesc(children.filter(e => e.kind == NOTE))
+
   $: disableActions = !$signer || (muted && !showHidden)
-  $: like = likes.find(e => e.pubkey === $sessionWithMeta?.pubkey)
+  $: liked = likes.find(e => e.pubkey === $sessionWithMeta?.pubkey)
   $: $likesCount = likes.length
-  $: zap = zaps.find(e => e.request.pubkey === $sessionWithMeta?.pubkey)
+  $: zapped = zaps.find(e => e.request.pubkey === $sessionWithMeta?.pubkey)
   $: $zapsTotal = sum(pluck("invoiceAmount", zaps)) / 1000
-  $: canZap = zapper?.allowsNostr && note.pubkey !== $sessionWithMeta?.pubkey
-  $: reply = replies.find(e => e.pubkey === $sessionWithMeta?.pubkey)
+  $: canZap = $zapper?.allowsNostr && event.pubkey !== $sessionWithMeta?.pubkey
+  $: replied = replies.find(e => e.pubkey === $sessionWithMeta?.pubkey)
   $: $repliesCount = replies.length
-  $: handlers = $kindHandlers.filter(
-    h =>
-      h.name.toLowerCase() !== "coracle" &&
-      h.event.tags.some(
-        t =>
-          ["web", os].includes(t[0]) && (t.length == 2 || ["note", "nevent", ""].includes(last(t))),
-      ),
-  )
+  $: handlers =
+    event.kind != 1 &&
+    $kindHandlers.filter(
+      h =>
+        h.name.toLowerCase() !== "coracle" &&
+        h.event.tags.some(
+          t =>
+            ["web", os].includes(t[0]) && (t.length == 2 || ["note", "nevent"].includes(last(t))),
+        ),
+    )
```

Comment:
There are important changes in this conflict, the empty mark is needed for certain handlers.

Reply:
blaming my pink eye for it